### PR TITLE
Add new sample for Cloud Run custom domain mapping

### PR DESCRIPTION
**Pull request by rogerthatdev**
To contribute to the following: https://cloud.google.com/run/docs/mapping-custom-domains

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
google_cloud_run_service: add sample for cloud run custom domain mapping

```

### CODE REVIEW

**Comment by slevenick** [comment]
It looks like the test is failing due to the domain mapping not being verified:
`Error checking ownership for domain tf-test-verified-domainyigz9u2yn2: Invalid Link in request: source 	 { domain_name { hostname: "tf-test-verified-domainyigz9u2yn2" } }`

It looks like the hostname is specified in the wrong format?

**Review by slevenick — CHANGES_REQUESTED** [review]
See my comment above about the test failing

**Comment by rogerthatdev** [comment]
Regardless of the format, I think that the domain used for this field needs to be a domain that's been verified owned by the organization.  Is there a test verified domain that can be used for this field?

File: mmv1/products/cloudrun/terraform.yaml [hunk 1]
```diff
@@ -316,6 +316,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         skip_docs: true
         vars:
           cloud_run_srv: "cloud-run-srv"
+        min_version: beta
```
Comment (slevenick) [inline mmv1/products/cloudrun/terraform.yaml:319]: This marks the previous example as beta-only. Is there a reason to do that?

Comment (rogerthatdev) [reply mmv1/products/cloudrun/terraform.yaml]: Not sure how it got there. I removed this line

**Comment by slevenick** [comment]
You may be able to use ORG_DOMAIN via getTestOrgDomainFromEnv in the test. I would guess that is a verified domain, but I don't have much experience using it.

It can be used like this: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/iap/terraform.yaml#L203

**Comment by c2thorn** [comment]
@rogerthatdev I believe you are already aware, but MM no longer generates CGC samples. Closing here to avoid confusion.

**Pull request closed without merging**